recipes/qutip/bld.bat

```diff
@@ -0,0 +1,8 @@
+conda clean --yes --source-cache
+
+"%PYTHON%" setup.py build --compiler=mingw32
+"%PYTHON%" setup.py build_ext --compiler=mingw32
+"%PYTHON%" setup.py install
+if errorlevel 1 exit 1
```

Comment:
@ajgpitch you need one `if errorlevel 1 exit 1` below each line or we won't know if the first two worked.